ocrd_tesserocr/binarize.py

```diff
@@ -52,10 +52,15 @@ def process(self):
         
         Produce a new output file by serialising the resulting hierarchy.
         """
+        try:
+            self.page_grp, self.image_grp = self.output_file_grp.split(',')
+        except ValueError:
+            self.page_grp = self.output_file_grp
+            self.image_grp = FILEGRP_IMG
```

Comment:
I really like this fallback. But we could give a warning here that images are going there.

Reply:
Done in 6c0ee6d